uniflow-core/src/main/kotlin/io/uniflow/core/flow/data/ThrowableKt.kt

```diff
@@ -0,0 +1,40 @@
+package io.uniflow.core.flow.data
+
+fun Throwable.toThrowableKt(): ThrowableKt {
+    return ThrowableKt(message, cause)
+}
+
+fun throwableKt(t: Throwable) = t.toThrowableKt()
+
+open class ThrowableKt(val message: String? = null, val cause: ThrowableKt? = null) {
```

Comment:
I'm not fond of the name.

Reply:
yep, would need a better name somewhere 🤔

Reply:
UniflowException

Reply:
`UniThrowable` 😹 😂 😹 (just kidding)

Reply:
`UIError` ? :)